app/assets/stylesheets/blocks/footer.scss

```diff
@@ -0,0 +1,8 @@
+footer {
+  border-top: 1px solid $dark-gray;
+
+  a {
+    color: $dark-gray;
+    line-height: 2rem;
+  }
+}
```

Comment:
Another "In Fincap ...." comment 😉We have things like this in a `_base.scss` file, which I think is quite good for element-level styles.

Our structure might be worth a look for you: https://github.com/boost/fincap-client-voices/tree/master/app/assets/stylesheets

Reply:
Good point, changed it :)